src/FANET/FanetStation.hpp

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2016 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#ifndef XCSOAR_FANET_STATIONS_HPP
+#define XCSOAR_FANET_STATIONS_HPP
+
+#include "util/StaticString.hxx"
+#include <type_traits>
+#include <tchar.h>
+#include "FanetAddress.hpp"
+#include "NMEA/Validity.hpp"
+#include "Geo/GeoPoint.hpp"
+
+struct FanetStation {
+  /** Station info */
+  float temperature;
+  float wind_dir_deg;
+  float wind_speed_kmph;
+  float wind_gust_kmph;
+  float rel_humidity;
+  double preasure_hpa;
+  float soc_percent;
```

Comment:
These need to be documented.
The variable names suggest that you're using non-standard units; that is error prone. XCSoar should use SI units everywhere, and special classes where we have them.
For pressure, we have a the `class AtmosphericPressure`. For angles, we have `Angle`.